app/views/records/index.html.erb

```diff
@@ -21,38 +21,48 @@
 
 <% else %>
   <h1>Master Patient List</h1>
-  <div class="pull-right button-row">
-    <%= button_to download_mpl_records_path(@bundle), :method => :get, :class => "btn btn-default" do %>
-      <i class="fa fa-fw fa-download" aria-hidden="true"></i> Download Master Patient List
-    <% end %>
-  </div>
 
   <div class="row">
-    <div class="col-md-7 col-sm-8">
+    <div class="col-md-5 col-sm-6">
       <h1>Filter Patients</h1>
-      <div class="form-group col-md-8 col-sm-10">
+      <div class="form-group">
         <label for="search_measures">Type to search by measure</label>
         <input id="search_measures" type="search" class="form-control" placeholder="Type the name or CMS ID of a measure"/>
       </div>
     </div>
 
     <% if !@task && !@product_test %>
-      <div class="col-md-5 col-sm-4">
+      <div class="col-md-7 col-sm-6">
         <h1>Annual Update Bundle</h1>
-        <% if Bundle.count == 1 %>
-          <!-- label --> <%= @bundle.title %>
-        <% else %>
-          <!-- loop through bundles & use radio buttons -->
-          <%= bootstrap_form_tag do |f| %>
-          <%= f.form_group :bundle_id do %>
+        <div class="col-md-8">
+          <% if Bundle.count == 1 %>
+          <div class="radio" style="padding: 6px 12px;">
+            <!-- label --> <%= @bundle.title %>
+          </div>
+          <% else %>
```

Comment:
Download buttons probably want to be vertically centered on the label

![screen shot 2016-11-07 at 2 19 17 pm](https://cloud.githubusercontent.com/assets/8173551/20072149/4bcca18c-a4f5-11e6-9bec-99b3968751db.png)

Here is an extreme example

![screen shot 2016-11-07 at 2 19 05 pm](https://cloud.githubusercontent.com/assets/8173551/20072161/554349b4-a4f5-11e6-85ee-907d46fa3f72.png)